modules/compute-vm/main.tf

```diff
@@ -333,6 +334,14 @@ resource "google_compute_instance" "default" {
       resource_manager_tags = local.tags_combined
     }
   }
+
+  dynamic "guest_accelerator" {
```

Comment:
is this also supported for instance templates? if so the same block needs to be added to that resource